kolibri/core/auth/tasks.py

```diff
@@ -809,8 +819,8 @@ def validate(self, data):
     queue=soud_sync_queue,
     permission_classes=[IsSuperAdmin() | NotProvisioned()],
 )
-def peeruserimport(**kwargs):
-    call_command("sync", **kwargs)
+def peeruserimport(command, **kwargs):
```

Comment:
Did this raise an error with no arg here?

Reply:
Yep